src/lib/db/project-store.ts

```diff
@@ -159,7 +155,7 @@ class ProjectStore implements IProjectStore {
             .where({
                 project_id: id,
             })
-            .returning('environment_name');
+            .pluck('environment_name');
```

Comment:
TIL we have pluck :+1:

Reply:
was new to me too. Had to find a fix as this one was not returning what we wanted and I start exploring the docs and discovered it by chance. I think it was added to knex recently.